### PR TITLE
epr: init at 2.3.0b

### PR DESCRIPTION
**Pull request by Br1ght0ne**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---
This change is [Reviewable](https://reviewable.io/reviews/nixos/nixpkgs/73038)

### CODE REVIEW

File: pkgs/applications/misc/epr/default.nix [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "1a6md3015284hzmx0sby5kl59p7lwv73sq7sid35vrr15zrl0aw7";
+  };
+
+  meta = with lib; {
+    description = "CLI Epub Reader";
+    homepage = "https://github.com/wustho/epr";
+    license = licenses.mit;
+    maintainers = [ ];
```
Comment (zimbatm) [inline pkgs/applications/misc/epr/default.nix]: hi, can you add yourself as a maintainer?

File: pkgs/applications/misc/epr/default.nix [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{ lib, python3Packages, fetchFromGitHub, ncurses }:
```
Comment (zimbatm) [inline pkgs/applications/misc/epr/default.nix]: ncurses doesn't seem to be used in the package description?

Comment (zimbatm) [reply pkgs/applications/misc/epr/default.nix]: It looks like python is already finding the curses package

File: pkgs/applications/misc/epr/default.nix (hunk 1 above)
Comment (filalex77) [reply pkgs/applications/misc/epr/default.nix]: Would be great, but I already have a PR adding me to the maintainer list (for `ruplacer`). Is it okay for me to duplicate these changes here?

File: pkgs/applications/misc/epr/default.nix (hunk 2 above)
Comment (filalex77) [reply pkgs/applications/misc/epr/default.nix]: Okay, thanks for pointing out!

File: pkgs/applications/misc/epr/default.nix (hunk 1 above)
Comment (zimbatm) [reply pkgs/applications/misc/epr/default.nix]: Either that or wait until ruplacer is merged in, both are fine

**Pull request merged**